app/models/concerns/outdateable.rb

```diff
@@ -7,8 +7,8 @@ module Concerns::Outdateable
     scope :outdated, -> { where(outdated: true, latest: true) }
     scope :outdated_before, ->(date) { outdated.where('outdated_at < ?', date) }
 
-    def outdate!
-      update_attributes!(outdated: true, outdated_at: Time.zone.now)
+    def outdate!(base_path:)
```

Comment:
I would not pass the `base_path` into the concern as it seems out of its main responsibility. This is going to change shortly once we start using `base_paths` as the natural key for Items, but for the time being, making the update explicit in https://github.com/alphagov/content-performance-manager/blob/87e84fb826f7b41d4b7d18d918622dcbb5e5f18d/lib/publishing_api_consumer.rb#L20-L21, along with an explanation in the commit notes would be beneficial.